### PR TITLE
Add GitHub Actions Maven CI

### LINKED ISSUE #2: Add Github Action CI for Maven Tests

### PR DESCRIPTION
**Pull request by twobiers**

### CODE REVIEW

**Review by vinamra28 — CHANGES_REQUESTED** [review]
could you please squash the commits :slightly_smiling_face:

**Comment by twobiers** [comment]
@vinamra28 done

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (vinamra28) [inline .github/workflows/maven.yml]: could you please setup this for opened and synchronized pull request as well?

Comment (vinamra28) [reply .github/workflows/maven.yml]: probably like adding 
```
  pull_request:
    types: [opened, synchronize]
```

Comment (twobiers) [reply .github/workflows/maven.yml]: Yes. Do You need any other type? 
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request

**Review by vinamra28 — APPROVED** [review]

File: .github/workflows/maven.yml (hunk 1 above)
Comment (vinamra28) [reply .github/workflows/maven.yml]: for now it's good. In next iteration maybe I'll work upon it.
Thanks for the PR :+1:

**Pull request merged**